frontend/js/src/mbid-mapping/MBIDMappingModal.tsx

```diff
@@ -207,16 +215,24 @@ export default NiceModal.create(({ listenToMap }: MBIDMappingModalProps) => {
                 feedbackComponent={<></>}
                 compact
               />
+
               <div className="text-center mb-10 mt-10">
-                <FontAwesomeIcon
-                  icon={faExchangeAlt}
-                  rotation={90}
-                  size="lg"
-                  color={
-                    selectedRecording ? COLOR_LB_GREEN : COLOR_LB_LIGHT_GRAY
-                  }
-                />
+                <button
+                  className="search-field-copy-button"
```

Comment:
After testing live, I think we should make it clearer that this is a clickable button. Currently I don't think it's obvious.
I tried with our existing css classes and settled on something.
In the end we don't need the custom `search-field-copy-button` class so I think those .less file changes will need to be removed.

<img width="483" alt="image" src="https://github.com/metabrainz/listenbrainz-server/assets/6179856/63b5cc27-78b2-42e5-be47-64328e9588d3">


The code changes are as follows:
```
<button
  className="btn btn-link btn-rounded"
…
```
And the additional text right under the FontAwesomeIcon:
```
<div class="text-muted">copy text</div>
```

Reply:
[Screencast from 13-09-23 11:07:53 PM IST.webm](https://github.com/metabrainz/listenbrainz-server/assets/44491059/8684dfed-b5cc-4d68-99dd-c815a54b9243)
When we hover over the button the text gets underlined and on clicking it the border gets highlighted, is this expected behavior?

Reply:
You're right, we can use `btn-transparent` instead of `btn-link`.
As for the highlighted border, looks like our other buttons have the same style from bootstrap, I'm guessing helpful for visually impared users